### PR TITLE
Caballlero patch 1

### PR DESCRIPTION
**Pull request by Caballlero**

### CODE REVIEW

**Comment by Caballlero** [comment]
It is my first time i am trying to help of this, if something are no correct let me know **geobra**

File: resources/qml/pages/ContactsPage.qml [hunk 1]
```diff
@@ -61,7 +61,7 @@ Page {
 
                 Label {
                     id: nameId;
-                    text: name
+                    text: qsTr("name")
```
Comment (geobra) [inline resources/qml/pages/ContactsPage.qml]: This is an id which gets replaced by c++ code with the contact name. Not translateable.

File: resources/qml/pages/ContactsPage.qml [hunk 2]
```diff
@@ -77,22 +77,22 @@ Page {
                     }
                     Label {
                         id: jidId;
-                        text: jid;
+                        text: qsTr("jid");
```
Comment (geobra) [inline resources/qml/pages/ContactsPage.qml]: This is an id which gets replaced by c++ code with the contact jabber id. Not translateable.

File: resources/qml/pages/ContactsPage.qml [hunk 3]
```diff
@@ -77,22 +77,22 @@ Page {
...
                         color: Theme.secondaryColor;
                         font.pixelSize: Theme.fontSizeTiny;
                     }
                 }
                 Label {
                     id: statusId;
-                    text: status;
+                    text: qsTr("status");
```
Comment (geobra) [inline resources/qml/pages/ContactsPage.qml]: This is the status which gets replaced by c++ code with the contact status. Not translateable.

File: resources/qml/pages/ConversationsPage.qml [hunk 4]
```diff
@@ -75,13 +75,13 @@ Page {
                 }
                 Label {
                     id: jidId;
-                    text: jid;
+                    text: qsTr("jid");
```
Comment (geobra) [inline resources/qml/pages/ConversationsPage.qml]: This is the jabber id which gets replaced by c++ code with the contact jid. Not translateable.

File: resources/qml/pages/ConversationsPage.qml [hunk 5]
```diff
@@ -75,13 +75,13 @@ Page {
...
                     color: Theme.secondaryColor;
                     font.pixelSize: Theme.fontSizeTiny;
                 }
                 Label {
                     id: statusId;
-                    text: lastmessage;
+                    text: qsTr("last message");
```
Comment (geobra) [inline resources/qml/pages/ConversationsPage.qml]: This is an id which gets replaced by c++ code with the last message of that contact. Not translateable.

**Pull request closed without merging**